src/Calypso-Browser/ClyQueryViewMorph.class.st

```diff
@@ -114,7 +114,8 @@ Class {
 		'highlighting',
 		'changesWasInitiatedByUser',
 		'navigationSelector',
-		'dataSourceClass'
+		'dataSourceClass',
+		'expandedItem'
```

Comment:
It would be better to have more concrete name like expandedItemByDrag. 

But generally this state needs to be managed together with other expansion state like tree expansion in class view for example. 
ClyDataSource is responsible for this. And it handles the cases where expanded items are updated (when they are removed for example). Maybe it does not matter for drag scenario but I can see how the view can get inconsistent state and possibly some errors can happen.

Reply:
From the other side I would expect different fix for the problem with "too fast drag". And I think there is existing issue with description:
Normally UI (non Pharo UI) requires to hang the drag item for a while over the drop target to force the expansion. Now it is expanded immediately. And that's what we need to change.

Reply:
I completely agree with everything you said, and that was my initial objective.
Buuuuuuut, I wasn't able to do it :/
This fix is not meant to be a definitive one, just to have an easier time dealing with drag and drops.

I'll do the rename though thanks ! =)